astropy/io/fits/convenience.py

```diff
@@ -408,9 +410,11 @@ def writeto(filename, data, header=None, output_verify='exception',
         ``+warn``, or ``+exception" (e.g. ``"fix+warn"``).  See :ref:`verify`
         for more info.
 
-    clobber : bool, optional
-        If `True`, and if filename already exists, it will overwrite
-        the file.  Default is `False`.
+    overwrite : bool, optional
+        If ``True``, overwrite the output file if exists.
```

Comment:
not exactly sure but the `,` seems weird. Also it should note that if `False` and the file exists and OSError is raised.